src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/ListedHearingService.java

```diff
@@ -277,12 +277,12 @@ private HearingCentre getHearingCentre(ServiceData serviceData) {
             : HearingCentre.getHearingCentreByEpimsId(getHearingVenueId(serviceData));
     }
 
-    private boolean isRemoteHearing(ServiceData serviceData) {
+    public boolean isRemoteHearing(ServiceData serviceData) {
         final String nextHearingChannel = getHearingChannels(serviceData).get(0).name();
```

Comment:
no unit test? it is now public method, it is better to have a unit test at least for test coverage